chain/src/main/java/org/ethereum/beacon/chain/storage/AbstractChainStorage.java

```diff
@@ -0,0 +1,47 @@
+package org.ethereum.beacon.chain.storage;
+
+import org.ethereum.beacon.chain.storage.BeaconBlockStorage;
+import org.ethereum.beacon.chain.storage.BeaconStateStorage;
+import org.ethereum.beacon.chain.storage.BeaconTupleStorage;
+import org.ethereum.beacon.chain.storage.ChainStorage;
+import org.ethereum.beacon.chain.storage.impl.BeaconTupleStorageImpl;
+import org.ethereum.beacon.db.Database;
+
+public abstract class AbstractChainStorage implements ChainStorage {
```

Comment:
Doesn't look like something abstract.
Though I'm ok to keep naming simple if it's in specific package.